test/src/com/xilinx/rapidwright/edif/TestEDIFPort.java

```diff
@@ -66,4 +67,49 @@ public void testEDIFPortInternalNets() {
             }
         }
     }
+
+    @Test
+    public void testCreatePort() {
```

Comment:
Does this test all the different collision scenarios we can think of? If so, can you enumerate them as a comment so we don't forget about them?

Reply:
I think with the convention change of adding the `[` suffix to all bussed ports, it alleviates the previous collision challenges.  However, I can imagine that we now have theoretical issues with port names such as `single_bit_port[` and `single_bit_port[[3:0]`.  Do we want to build out support for these?

Reply:
> However, I can imagine that we now have theoretical issues with port names such as `single_bit_port[` and `single_bit_port[[3:0]`. Do we want to build out support for these?

Absolutely not! 

I just couldn't figure out if this test tests any collisions, so maybe a quick comment to state which collisions are being tested. Off the top of my head:

`foo` vs `foo[0]` -- both bitty
`foo[0]` vs `foo[1]` -- both bitty
`foo[0]` vs `foo[0][0]` -- both bitty
`foo[0]` vs `foo[0][7:0]` -- one bitty one bussy

Anything else? Particularly the scenario that the current implementation already resolves, plus the new scenario that you discovered recently which necessitated this rewrite.

Reply:
I don't think there are really any others that we want to test for.  I've updated the test to enumerate and test for these.